platform/security/filter/security-filter-csrf/src/main/java/org/codice/ddf/security/filter/csrf/CsrfFilter.java

```diff
@@ -55,15 +57,42 @@ public void init(FilterConfig filterConfig) throws ServletException {
     protectedContexts.add(JOLOKIA_CONTEXT);
     protectedContexts.add(INTRIGUE_CONTEXT);
     protectedContexts.add(WEBSOCKET_CONTEXT);
+
+    // internal authority system properties
+    String internalHostname = System.getProperty("org.codice.ddf.system.hostname");
```

Comment:
✏️ Can we use the `SystemBaseUrl` getters instead of the property literals? It prevents typos and if we ever need to refactor any of these again it is *much* easier to catch.